### PR TITLE
Public preview for MDE behind proxy.

### PR DESCRIPTION
**Pull request by idbasre**

### CODE REVIEW

**Comment by idbasre** [comment]
@dcurwin I see I have a warning for using an absolute link. How can I see the relative link? (it is in MDE documentation)
This is the link:
https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/microsoft-defender-antivirus-windows?view=o365-worldwide#comparing-active-mode-passive-mode-and-disabled-mode

File: CloudAppSecurityDocs/mde-integration.md [hunk 1]
```diff
@@ -31,6 +31,9 @@ Once traffic information is collected, you're ready to [deep dive into cloud app
   - [Cloud-delivered protection enabled](/microsoft-365/security/defender-endpoint/enable-cloud-protection-microsoft-defender-antivirus)
   - [Network protection enabled and configured to block mode](/microsoft-365/security/defender-endpoint/enable-network-protection)
 
+> [!NOTE]
+The ability to perform Cloud Disocvery when Defender for Endpoint is deployed behind a proxy is in public preview. To do so, please make sure Defender for Endpoint is in **Active** mode (see here how to check it: [Microsoft Defender Antivirus in Windows | Microsoft Docs](https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/microsoft-defender-antivirus-windows?view=o365-worldwide#comparing-active-mode-passive-mode-and-disabled-mode)).
```
Comment (dcurwin) [inline CloudAppSecurityDocs/mde-integration.md:35]:
```suggestion
The ability to perform Cloud Discovery when Defender for Endpoint is deployed behind a proxy is in public preview. To use it, make sure Defender for Endpoint is in **Active** mode. as explained here: [Comparing active mode, passive mode, and disabled mode](/microsoft-365/security/defender-endpoint/microsoft-defender-antivirus-windows#comparing-active-mode-passive-mode-and-disabled-mode).
```

**Comment by dcurwin** [comment]
Learn Build status updates of commit _[447b283](https://github.com/idbasre/CloudAppSecurityDocs/commits/447b28340bfbd9201b6ddf8b094d57ad67d2e2d6)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[CloudAppSecurityDocs/mde-integration.md](https://github.com/idbasre/CloudAppSecurityDocs/blob/patch-40/CloudAppSecurityDocs/mde-integration.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/daf9759b-aa27-219c-4b5e-53f1b72a2d34/202211160933118257-612/BuildReport?accessString=026702a49fb98e318c2f5e94b4a504064a9e968c69599e0de24eb0de2ee043c5).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/daf9759b-aa27-219c-4b5e-53f1b72a2d34?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

**Comment by dcurwin** [comment]
Learn Build status updates of commit _[447b283](https://github.com/idbasre/CloudAppSecurityDocs/commits/447b28340bfbd9201b6ddf8b094d57ad67d2e2d6)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[CloudAppSecurityDocs/mde-integration.md](https://github.com/idbasre/CloudAppSecurityDocs/blob/patch-40/CloudAppSecurityDocs/mde-integration.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/daf9759b-aa27-219c-4b5e-53f1b72a2d34/202211161045152324-612/BuildReport?accessString=79f4b2b9f4d3a84baf7ed760c244db063f44aa757ea8b9ceae8c68c501575e4f).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/daf9759b-aa27-219c-4b5e-53f1b72a2d34?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

**Pull request closed without merging**